src/roadmap/v1/upcoming.py

```diff
@@ -1,12 +1,76 @@
+from datetime import date
+from enum import auto
+from enum import StrEnum
+from functools import lru_cache
+from pathlib import Path
+from typing import Annotated
+from typing import Any
```

Comment:
Just import `typing` once and then you can reference any type without having to do additional imports.
```suggestion
import typing as t
```

Reply:
fixed